src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -34,11 +48,11 @@ public function __construct(ObjectManager $om, $doFlush = true)
     public function persist(array $objects)
     {
         foreach ($objects as $object) {
-            $this->om->persist($object);
+            $this->manager->persist($object);
         }
 
-        if ($this->flush) {
-            $this->om->flush();
+        if (true === $this->flush) {
```

Comment:
Not sure that this is really necessary, since `$this->flush` isn't used in a way where truthy / falsy values would cause problems.